app/views/pages/moving-guide/tips.html.erb

```diff
@@ -9,11 +9,11 @@
     <p>Whether this is your first move or you’re a seasoned expert… moving is stressful. While the hope would be that everything goes off without a hitch, we’ve combed service members and their families far and wide to compile some of the Top Tips to help you out.</p>
     <p>This list certainly is not the end all be all, but it’s a great place to start!</p>
 
-    <div class="tabs">
+    <div class="tabs" id="tabs">
```

Comment:
These ids are required for the correct tab to be selected on page load when the URL includes an anchor pointing to one of the tabs.

Reply:
I think there's a bigger issue in the JavaScript (that I introduced… 😑 ) for the tabs that needs addressing. I vote we leave this change out and file an issue to take another look at that script.

Reply:
All I did was make the ids on tabs on pages/moving-guide/tips.html.erb match the tabs on service_specific_information/index.html/erb. This change makes the tips page behave correctly, just like the service-specific page already did. I included the change in this PR because there are links on the FAQ page which point to specific tabs on the tips page.

If there's a problem with tabs.js, then sure, we can file an issue about it, but I must be missing what's actually incorrect about the change.